client/src/main/java/org/eclipse/hono/client/MessageSender.java

```diff
@@ -8,22 +8,25 @@
  *
  * Contributors:
  *    Bosch Software Innovations GmbH - initial creation
+ *
  */
 
 package org.eclipse.hono.client;
 
 import java.util.Map;
+import java.util.function.BiConsumer;
 
 import org.apache.qpid.proton.message.Message;
 
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Handler;
+import io.vertx.proton.ProtonDelivery;
 
 /**
  * A client for uploading telemetry data to a Hono server.
  *
  */
-public interface TelemetrySender {
+public interface MessageSender {
 
     /**
      * Checks if this sender can send or buffer (and send later) a telemetry message.
```

Comment:
s /`telemetry message`/`message`?  

this applies to all JavaDoc in this interface ...